src/s1/s1/static/js/Packages/S1/Workbench/ProposalEditColumn.html

```diff
@@ -3,7 +3,7 @@
         <div class="moving-column-menu-nav">
             <a
                 class="moving-column-menu-nav-button"
-                data-ng-href="{{ processUrl | adhResourceUrl:meeting }}">{{ 'TR__CLOSE' | translate }} X</a>
+                data-ng-href="{{ proposalUrl | adhResourceUrl }}">{{ 'TR__CLOSE' | translate }} X</a>
```

Comment:
I don't understand this change. The close button is supposed to close the column.